frontend/src/app/spot/styles/sass/components/link.sass

```diff
@@ -1,7 +1,7 @@
 .spot-link
   @include unset-button-styles
   display: inline
-  color: $spot-color-main
+  color: var(--content-link-color)
```

Comment:
This is not enough as it brings us back to the problem we originally wanted to solve with the new links. For the default theme (as well as the light theme by the way) the `content-link-hover-active-color` is the same as `content-link-color` resulting in no visible hover effect except the underlining. So with the theme colours we need the underlining as well.

Reply:
Hi @b12f and @HDinger,

Sorry for the confusion on the mobile modal again! We where doing some trials with the Product Team on how we could harmonise the buttons in all the modals in the complete application. Please don't actualise anything in regards of this full width button on the mobile version. Also there is an error also solved where the clear selection button says cancel, please keep the clear selection option.